src/actions/TemperatureSyncActions.js

```diff
@@ -77,7 +80,15 @@ const scanForSensors = () => async dispatch => {
 
   dispatch(updateSensors(data));
 
-  if (success) dispatch(scanComplete());
+  if (success) {
+    ToastAndroid.show(
+      vaccineStrings.formatString(vaccineStrings.found_sensors, data.length),
+      ToastAndroid.LONG
+    );
+    dispatch(scanComplete());
+  } else {
+    ToastAndroid.show(vaccineStrings.could_not_find_sensors, ToastAndroid.LONG);
```

Comment:
I don't like the toast here. I really want to create a `<Toast/>` component which sits at the root which can be decoratively show a toast - but not 100% sure how I should do it and didn't want to do it here.

Reply:
but this action was created previously - just uses `Sensor.scanForSensors` which initializes the bluetooth module and scans for any advertisement frequencies from sensors with the right manufacturer ID and characteristics and returns an object for each found sensor with all the info from its advertisement - battery level, mac address etc. Uses another action `updateSensors` with that data to actually update sensors. Toasts when finding sensors with how many and if none were found, says that. The `startScan` and `completeScan` set the state to scanning: true/false, which will also stop the `TemperatureSyncModal` from being enabled and trying to start another scan - also updates the modal state so if you open the modal it will say "scanning for sensors"

Reply:
I like the idea of dispatching a `showToast` action and showing the toast from a reducer just because that's a similar process to other UI display tasks. Fair enough to leave the 🍞 changes to a separate issue though, and this is fine like it is.